test/suite.go

```
@@ -59,3 +59,27 @@
+	defer cancel()
+
+	stateRoot, maxBytes, err := s.Exec.InitChain(ctx, genesisTime, initialHeight, chainID)
+	s.Require().NoError(err)
+	s.NotEqual(types.Hash{}, stateRoot)
+	s.Greater(maxBytes, uint64(0))
+
+	for i := initialHeight; i <= 10; i++ {
+		txs, err := s.Exec.GetTxs(ctx)
+		s.Require().NoError(err)
+
+		stateRoot, maxBytes, err = s.Exec.ExecuteTxs(ctx, txs, i, time.Now(), stateRoot)
```

golangci-lint (1.62.2)

ineffectual assignment to maxBytes

(ineffassign)